README.md

```diff
@@ -87,6 +87,19 @@ eql is an Erlang library with similar inspiration and goals.
 - A subjective point, but I believe the Yesql's implementation is simpler than
   eql's, while offering more features.
 
+### [ayesql](https://github.com/alexdesousa/ayesql)
+
+ayesql is another Elixir library, a bit more powerful than yesql:
+
+- It offers support for various SQL statements in a single file.
+- Special constructs for [query composability](https://hexdocs.pm/ayesql/readme.html#query-composition) in SQL files.
+- Special constructs for [optional parameters](https://hexdocs.pm/ayesql/readme.html#optional-parameters) in SQL files.
+
+Should I use ayesql then? Not necessarily... it may depend on your style preference.
```

Comment:
Unsure about the wording here, it may not even be a "style preference".

Reply:
I think we can remove this line andusrs can decide for themselves